integration/helpers/cf-template/app/root.tsx

```diff
@@ -1,5 +1,5 @@
-import type { LinksFunction } from "@remix-run/cloudflare";
-import { cssBundleHref } from "@remix-run/css-bundle";
+import type { LinksFunction } from "@react-router/cloudflare";
+import { cssBundleHref } from "@react-router/css-bundle";
```

Comment:
I think the css-bundle package is obsolete in a `vite`-only world?

Reply:
We have a few loose ends like this lying around, so I think we can merge this as-is for now and do a broader cleanup later.